app/src/main/java/com/eveningoutpost/dexdrip/CalibrationGraph.java

```diff
@@ -114,4 +126,111 @@ public Line getCalibrationsLine(List<Calibration> calibrations, int color) {
         line.setHasLabels(true);
         return line;
     }
+
+
+
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(xdrip.getAppContext());
+
+
+        //Just generate the menu in engineering mode
+        if (!prefs.getBoolean("engineering_mode", false)){
+            return false;
+        }
+
```

Comment:
There is a function for it in xdrip-plus: home.getPreferencesBooleanDefaultFalse()